package.json

```diff
@@ -120,6 +120,7 @@
     "markdown-spellcheck": "^1.3.1",
     "merge-stream": "^2.0.0",
     "merge2": "^1.2.3",
+    "mini-css-extract-plugin": "^0.6.0",
```

Comment:
Was this added because webpack 4 requires it?

Reply:
By the looks of it yes as for the `webpack.config.babel.js` file moves from `ExtractTextPlugin` to use `MiniCssExtractPlugin` plugin instead